src/antelope/wallets/utils/currency-utils.ts

```diff
@@ -129,7 +129,7 @@ export function prettyPrintCurrency(
     }
 
     const decimalSeparator = getDecimalSeparatorForLocale(locale);
-    const trailingZeroesRegex = new RegExp(`\\${decimalSeparator}?0+(\\D|$)`, 'g');
+    const trailingZeroesRegex = new RegExp(`(\\d)\\${decimalSeparator}0+(\\D|$)`, 'g');
```

Comment:
fix an issue with `prettyPrintCurrency` where `0.0000` is formatted as an empty string, when trim zeroes is true